project-tracker backend/src/app/routers/user.ts

```diff
@@ -1,10 +1,13 @@
 import { Router } from 'express';
 import * as userController from '../controllers/user';
+import * as userProjectController from "../controllers/userProject";
 
 export const router: Router = Router({ mergeParams: true });
 
 router.get('/', userController.index);
 router.get('/:id', userController.show);
 router.post('/', userController.create);
 router.put('/:id', userController.update);
-router.delete('/:id', userController.destroy);
\ No newline at end of file
+router.delete('/:id', userController.destroy);
+
+router.get('/:userId/project', userProjectController.index);
```

Comment:
ennek nem itt a helye, hanem külön fájlban

Reply:
gondoltam, ha már a /user/... közös, lehetnének egy router-ben is

Reply:
de ne megmagyarázd kérlek, hanem csináld úgy, ahogy mondom. Majd ha te leszel a lead, egy fájlba rakod, a kis tasik meg méltatlankodnak miatta.

Reply:
jelen idő szerint több, mint 48 órája javítva van, a kérésed szerint